react/features/shared-video/components/web/SharedVideoButton.js

```diff
@@ -0,0 +1,144 @@
+// @flow
+
+import type { Dispatch } from 'redux';
+
+import { isSharingStatus } from '../../';
```

Comment:
can you make this import explicit please?

Reply:
Done.